targets/drive/manifest.webapp

```diff
@@ -1,21 +1,39 @@
 {
   "name": "Drive",
   "slug": "drive",
+  "type": "webapp",
   "icon": "public/app-icon.svg",
-  "description": "File manager for Cozy v3",
-  "category": "cozy",
-  "source": "https://github.com/cozy/cozy-drive.git@build-drive",
+  "categories": ["cozy"],
+  "source": "https://github.com/cozy/cozy-drive",
   "editor": "Cozy",
+  "name_prefix": "Cozy",
   "developer": {
     "name": "Cozy",
     "url": "https://cozy.io"
   },
-  "default_locale": "en",
   "locales": {
+    "en": {
+      "short_description": "Cozy Drive is the app to save, sync and secure your files on your Cozy",
+      "long_description": "With Cozy Drive, you can:\n- Save and sync all your important files for free (passport, holiday photos, tax declarations, payslips…)\n- Access your documents any place, any time or even in flight mode from your office, smartphone or tablet;\n- Share links to files ans folders with who you like;\n- Automatically retrieve administrative documents from your main service providers (internet and phone providers, insurers, health professionals, …)\n- Stay in sync during your travels with our mobile apps",
```

Comment:
> Access your documents any place, any time or even in flight mode from your office, smartphone or tablet

_« Access your documents anytime, anywhere or even in plane mode from your desktop, smartphone or tablet »_